### DISCUSSION TITLE
Import connections from SQLDeveloper

### DISCUSSION THREAD

**Discussion by Ferrox12** [dbeaver/dbeaver#16742] [Q&A] [answered]
Hi,
is there a method to import all the connections and the related passwords from SQLDeveloper to DBeaver?

**Comment by Ferrox12** [comment]
I've found the function to import the connections!

**Comment by LonwoLonwo** [comment]
Can you provide some instructions here please @Ferrox12 

It can be helpful for other users with this question.

**Comment by Ferrox12** [comment, marked as answer]
You have to click on File > Import > Third Party Configuration > Oracle SQL Developer

**Reply by dineshbhagat** [reply]
This does not import connection passwords from file exported by sql Developer
